c7n/tags.py

```diff
@@ -1038,7 +1038,7 @@ def process(self, resources):
 
         stats = Counter()
 
-        for related, r in related_resources.items():
+        for related, r in related_resources:
```

Comment:
for the not found case, afaics from the diff (might not be needed if the rest of code already handles), this also needs
```
if not related:
   continue
```

Reply:
the following line should catch when it checks `related in missing_related_tags`